<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::DotLine</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Groups</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>41</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Group Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Group Type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>450</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>300</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>NEW</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>DELETE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>SAVE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>CLOSE</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFirm"/>
   <addaction name="actionValuer"/>
   <addaction name="actionGroups"/>
   <addaction name="actionItems"/>
   <addaction name="actionMarket_Rates"/>
   <addaction name="actionValuation"/>
   <addaction name="actionSplit_Valuation"/>
   <addaction name="actionChange_Password"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionFirm">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-organization-50.png</normaloff>:/icons/icons8-organization-50.png</iconset>
   </property>
   <property name="text">
    <string>Firm</string>
   </property>
   <property name="toolTip">
    <string>Firm</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionValuer">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-user-50.png</normaloff>:/icons/icons8-user-50.png</iconset>
   </property>
   <property name="text">
    <string>Valuer</string>
   </property>
   <property name="toolTip">
    <string>Valuer</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="actionGroups">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-list-50.png</normaloff>:/icons/icons8-list-50.png</iconset>
   </property>
   <property name="text">
    <string>Groups</string>
   </property>
   <property name="toolTip">
    <string>Groups</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
  <action name="actionItems">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-ring-50.png</normaloff>:/icons/icons8-ring-50.png</iconset>
   </property>
   <property name="text">
    <string>Items</string>
   </property>
   <property name="toolTip">
    <string>Items</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionMarket_Rates">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-us-dollar-50.png</normaloff>:/icons/icons8-us-dollar-50.png</iconset>
   </property>
   <property name="text">
    <string>Market Rates</string>
   </property>
   <property name="toolTip">
    <string>Market Rates</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionValuation">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-contract-50.png</normaloff>:/icons/icons8-contract-50.png</iconset>
   </property>
   <property name="text">
    <string>Valuation</string>
   </property>
   <property name="toolTip">
    <string>Valuation</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-password-50.png</normaloff>:/icons/icons8-password-50.png</iconset>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
   <property name="toolTip">
    <string>Change Password</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-about-50.png</normaloff>:/icons/icons8-about-50.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/icons/icons8-exit-50.png</normaloff>:/icons/icons8-exit-50.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSplit_Valuation">
   <property name="icon">
    <iconset resource="../../Evaluation Report Software/Valuation Main/toolbar.qrc">
     <normaloff>:/icons/icons8-separate-document-50.png</normaloff>:/icons/icons8-separate-document-50.png</iconset>
   </property>
   <property name="text">
    <string>Split Valuation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="toolbar.qrc"/>
  <include location="../../Evaluation Report Software/Valuation Main/toolbar.qrc"/>
 </resources>
 <connections/>
</ui>
